plugins/usd/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.19)
+
+project(f3d-plugin-usd)
+
+include(GNUInstallDirs)
+
+# Check if the plugin is built externally
+if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
+  find_package(f3d REQUIRED)
+else()
+  include(${CMAKE_SOURCE_DIR}/cmake/f3dPlugin.cmake)
+endif()
+
+find_package(pxr REQUIRED)
```

Comment:
what is pxr ?

Reply:
"pixar" I guess?

Reply:
so we depend on pxr ? does usd depend on it ?

Reply:
USD package is pxr. I know it's weird but that's how the CMake package is called.

Reply:
It all makes sense now ^^